config/default/webform.settings.yml

```diff
@@ -139,7 +139,6 @@ element:
   default_algolia_places_app_id: ''
   default_algolia_places_api_key: ''
   excluded_elements:
-    captcha: captcha
```

Comment:
Not sure why this exported or what it means.

Reply:
makes me think that captcha is now an allowed element.